extensions/ql-vscode/package.json

```diff
@@ -1402,7 +1403,7 @@
       "prettier --write"
     ],
     "./**/*.{ts,tsx}": [
-      "tsfmt -r",
+      "prettier --write",
       "eslint --fix"
```

Comment:
I don't think this is actually possible; `lint-staged` will run these commands with the filenames as arguments. If we had changes only the files `a.ts` and `b.ts`, this would run:

```shell
prettier --write a.ts b.ts
eslint --fix a.ts b.s
```

Using `npm run format`, it would run:

```shell
npm run format a.ts b.ts
```

This would in turn run:

```shell
prettier --write src/**/*.{ts,tsx} test/**/*.ts && eslint src test --ext .ts,.tsx --fix a.ts b.ts
```

This would not limit the files on which Prettier and ESLint are run, so in that case using `lint-staged` would not make sense.